Update v1.7 EOL for dbt Cloud Enterprise customers (#6367)
fishtown-analytics/docs.getdbt.com: website/snippets/core-versions-table.md

|                      dbt Core                                 | Initial release |      Support level and end date      |
|:-------------------------------------------------------------:|:---------------:|:-------------------------------------:|
| [**v1.8**](/docs/dbt-versions/core-upgrade/upgrading-to-v1.8) | May 9 2024    | <b>Active Support &mdash; May 8, 2025</b>                    |

> | [**v1.7**](/docs/dbt-versions/core-upgrade/upgrading-to-v1.7) | Nov 2, 2023   | Critical Support &mdash; EXTENDED for customers of dbt Cloud Enterprise<sup>1</sup> | 

Reviewer: ```suggestion
| [**v1.7**](/docs/dbt-versions/core-upgrade/upgrading-to-v1.7) | Nov 2, 2023   | dbt Core and dbt Cloud Developer/Team customers: Critical Support until Nov 1, 2024 <br/>
dbt Cloud Enterprise customers: Critical Support until further notice <sup>1</sup> | 
```

Reviewer: idk if you're allowed to embed `<br/>`s in a markdown table - guess I'll find out when it builds!

Reviewer 2: I believe you can, but watch it say I'm wrong :)

| [**v1.6**](/docs/dbt-versions/core-upgrade/upgrading-to-v1.6) | Jul 31, 2023  | End of Life ⚠️ |  
| [**v1.5**](/docs/dbt-versions/core-upgrade/upgrading-to-v1.5) | Apr 27, 2023  | End of Life ⚠️ |
| [**v1.4**](/docs/dbt-versions/core-upgrade/Older%20versions/upgrading-to-v1.4) | Jan 25, 2023  | End of Life ⚠️ | 
| [**v1.3**](/docs/dbt-versions/core-upgrade/Older%20versions/upgrading-to-v1.3) | Oct 12, 2022  | End of Life ⚠️ |
| [**v1.2**](/docs/dbt-versions/core-upgrade/Older%20versions/upgrading-to-v1.2) | Jul 26, 2022  | End of Life ⚠️ | 
| [**v1.1**](/docs/dbt-versions/core-upgrade/Older%20versions/upgrading-to-v1.1) | Apr 28, 2022  | End of Life ⚠️ |
| [**v1.0**](/docs/dbt-versions/core-upgrade/Older%20versions/upgrading-to-v1.0) | Dec 3, 2021   | End of Life ⚠️ | 
|  **v0.X** ⛔️                                               | (Various dates) | Deprecated ⛔️  | Deprecated ⛔️     | 

All functionality in dbt Core since the v1.7 release is available in dbt Cloud, early and continuously, by selecting ["Versionless"](https://docs.getdbt.com/docs/dbt-versions/versionless-cloud).

> Starting in November 2024, "Versionless" will be required for the Developer and Teams plans on dbt Cloud. After that point, accounts on older versions will be migrated to "Versionless."

Reviewer: ```suggestion
<sup>1</sup> Starting in November 2024, "Versionless" will be required for the Developer and Teams plans on dbt Cloud. After that point, accounts on older versions will be migrated to "Versionless."
```

|                      dbt Core                                 | Initial release |      Support level and end date      |
|:-------------------------------------------------------------:|:---------------:|:-------------------------------------:|
| [**v1.8**](/docs/dbt-versions/core-upgrade/upgrading-to-v1.8) | May 9 2024    | <b>Active Support &mdash; May 8, 2025</b>                    |

> | [**v1.7**](/docs/dbt-versions/core-upgrade/upgrading-to-v1.7) | Nov 2, 2023   | dbt Core and dbt Cloud Developer/Team customers: Critical Support until Nov 1, 2024\  

Reviewer 2: Let's try this
```suggestion
| [**v1.7**](/docs/dbt-versions/core-upgrade/upgrading-to-v1.7) | Nov 2, 2023   | dbt Core and dbt Cloud Developer/Team customers: Critical Support until Nov 1, 2024 <span><br /></span>
```

> | [**v1.7**](/docs/dbt-versions/core-upgrade/upgrading-to-v1.7) | Nov 2, 2023   | dbt Core and dbt Cloud Developer/Team customers: Critical Support until Nov 1, 2024 <br/>

Reviewer 2: ```suggestion
| [**v1.7**](/docs/dbt-versions/core-upgrade/upgrading-to-v1.7) | Nov 2, 2023   | dbt Core and dbt Cloud Developer/Team customers: Critical Support until Nov 1, 2024\  
```
